pyhacrf/tests/test_model.py

```diff
@@ -135,51 +117,8 @@ def test_fit_predict_regularized(self):
         assert_array_almost_equal(actual_predictions, expected_predictions, 
                                   decimal=TEST_PRECISION)
 
-    def test_fit_predict_regularized_viterbi(self):
```

Comment:
Viterbi code was never used, so I cleaned that up. I can deal with that in a separate PR if you would like.